### PR TITLE
Update code style of Guard auth

### PR DESCRIPTION
**Pull request by Saphyel**
I just converted array() into []

### CODE REVIEW

File: security/guard_authentication.rst [hunk 1]
```diff
@@ -221,27 +221,27 @@ This requires you to implement several methods::
             return null;
         }
 
-        public function onAuthenticationFailure(Request $request, AuthenticationException $exception)
+        public function onAuthenticationFailure(Request $request, AuthenticationException $exception): JsonResponse
```
Comment (ricknox) [inline security/guard_authentication.rst]: As far as I know the documentation, return type declarations are not used.
@javiereguiluz Can you give an opinion on this?

File: security/guard_authentication.rst [hunk 2]
```diff
@@ -221,27 +221,27 @@ This requires you to implement several methods::
...
+            ];
 
             return new JsonResponse($data, Response::HTTP_FORBIDDEN);
         }
 
         /**
          * Called when authentication is needed, but it's not sent
          */
-        public function start(Request $request, AuthenticationException $authException = null)
+        public function start(Request $request, AuthenticationException $authException = null): JsonResponse
```
Comment (ricknox) [inline security/guard_authentication.rst]: The same question as before, unsure if return type declarations are been used.

**Review by ghost — CHANGES_REQUESTED** [review]
Thank you for the inconsistency correction. However, in the Symfony Docs we still use the long array() syntax.

**Comment by xabbuh** [comment]
Thank you for your work @Saphyel. But unfortunately, @ricknox is right and I need to close your PR as we indeed still use the long array syntax in the docs. I hope to see you soon again with some other changes. :)

**Pull request closed without merging**